### ISSUE TITLE
Screenshot file location when running in runInstrumentation false mode

### ISSUE THREAD

**Issue by huwdfj** [Karumi/Shot#221]
Hi, I am trying to just run the screenshot comparison functionality i.e. runInstrumentation is set to false
When I run in this mode I get the following details in the console

"We couldn't find any screenshot. Did you configure Shot properly and added your tests to your project?"

Could you clarify where shot is expecting the screenshots to be in this scenario?

I presume the recorded screenshots should be in the project/screenshots/flavour/debug like a normal test run.
However I am not sure where I should place the new screenshots I am comparing. These would be the equivalent of the ones pulled from the device during a standard test run?

### Expected behaviour
During a test run with runInstrumentation = false, shot should find the screenshots for comparison

### Actual behaviour
I get the message
"We couldn't find any screenshot. Did you configure Shot properly and added your tests to your project?"

### Steps to reproduce
1. Run the tests and record the screenshots
2. Run a normal test run _executeScreenshotTests_ with runInstrumentation = true
3. This passes fine
4. Do another test run with runInstrumentation = false. All files still seem to be in the folder structure from the previous test run e.g. recorded images are under project/screenshots/flavour/debug and images from the previous test run are under /tmp/shot/screenshot/project
5. I get the message "We couldn't find any screenshot. Did you configure Shot properly and added your tests to your project?"

### Version of the library
5.10.4

**Comment by huwdfj** [comment]
I have now worked out the location based on the information from some previous issue.
You need to create a folder under project/screenshots/flavour/debug/screenshots-default

However if you create this folder and place images in there you now get passing tests regardless of what images are present and an empty comparison report with no images displayed?

![Screenshot 2021-04-21 at 17 20 56](https://user-images.githubusercontent.com/24966246/115587802-f9e93780-a2c5-11eb-82e6-3093da008bc4.png)

**Comment by SamCosta1** [comment]
I'm seeing exactly the same issue! Any pointers would be appreciated

**Comment by SamCosta1** [comment]
@huwdfj We fixed it!

TLDR: Make sure your metadata.xml file is included in your `screenshots-default` directory.

This is the file structure on the device:
![Screenshot 2021-04-23 at 16 04 45](https://user-images.githubusercontent.com/16070190/115891186-b588a380-a44d-11eb-9414-3ac1c776c56d.png)

The zip should be unzipped and placed in your `project/screenshots/{flavour}/debug`. The unzipped file should be renamed `screenshots-default`

Next take the `metadata.xml` and add it into the same `screenshots-default` directory.

Your final file structure should resemble the following
![Screenshot 2021-04-23 at 16 11 30](https://user-images.githubusercontent.com/16070190/115892090-bcfc7c80-a44e-11eb-924e-b6b250ddb8d4.png)

**Comment by huwdfj** [comment]
Hey @SamCosta1 are you obtaining the screenshots via Firebase Test Lab?
How are you creating the metadata.xml file?

**Comment by SamCosta1** [comment]
Ah right no we're not pulling from firebase test lab (though that is also our end goal). Currently this is just local testing.
The metadata file is generated by the library and is alongside the zip on the device. (See first screenshot).

**Comment by huwdfj** [comment]
Managed to pull the correct files by customising our test runner.
